### PR TITLE
Review of new French translation -- Working with VPCs

### PR DESCRIPTION
**Pull request by sweetsweetstefan**
#### Objective
I need a review of the French translation of this page: https://github.com/cloudops/cloudmc-standard-kb/blob/docs/MC-12361-working-with-vpcs/fr/cloudstack-compute-service/working-with-vpcs.md

### CODE REVIEW

File: fr/cloudstack-compute-service/working-with-vpcs.md [hunk 1]
```diff
@@ -4,58 +4,82 @@ slug: gestion-des-vpcs
...
+1. Si nécessaire, configurez l'autre extrémité du VPN avec la même clé pré-partagée que celle-ci.
+1. Une fois que l'autre extrémité du tunnel VPN a été configurée, l'état passera à **connecté**.
 
-##### Considérations :
+### Accès VPN à distance
 
-- Un sous-réseau du VPC ne doit pas interférer avec les adresses du site distant
-- Le VPN utilise le protocole IPsec
-- La sécurité du VPN est géré via des listes d’accès (ACL)
+Un VPN d'accès à distance vous permet d'accéder au réseau aux ressources d'un VPC. Pour plus d'informations, consultez le [Connexion à un VPC par une connexion VPN sécurisée](../vpn/cca-using-remote-access.md).
```
Comment (GuillaumeVialle) [inline fr/cloudstack-compute-service/working-with-vpcs.md:85]: remove `le` after consultez

File: fr/cloudstack-compute-service/working-with-vpcs.md [hunk 2]
```diff
@@ -4,58 +4,82 @@ slug: gestion-des-vpcs
...
+1. Sélectionnez **Services** dans la barre latérale gauche.
+1. Sélectionnez l'environnement de calcul approprié.
+1. Sélectionnez l'onglet **Réseautique**.
+1. Cliquez sur **Configurer la réseautique**, et dans le menu déroulant, choisissez **Ajouter un VPC**.
+![Onglet de réseautique](/assets/working-with-vpcs-1-fr.png)
+1. Complétez le formulaire d'ajouter un VPC :
+   - **Nom :** Nom du VPC (ex. *acme-prod-vpc01*)
+   - **Description :** Description du VPC (ex. "Site de production A").
+   - **CIDR :**  Plage d'adresses IP à utiliser pour le VPC. La plage doit être un réseau /22.
+   - **Domaine du réseau :** (Facultatif) Nom de domaine pour la résolution DNS interne (ex: *interne.acme.com*). CloudMC ajoutera ce nom de domaine au fichier */etc/hosts* pour les instances nouvelles.
```
[the remark this answers is not in the document]
Comment (GuillaumeVialle) [reply fr/cloudstack-compute-service/working-with-vpcs.md:27]: or `nouvelles instances`

File: fr/cloudstack-compute-service/working-with-vpcs.md [hunk 3]
```diff
@@ -4,58 +4,82 @@ slug: gestion-des-vpcs
...
 
-Un VPN site-à-site permet l’interconnection:
-
-   - De plusieurs VPCs
-   - De votre bureau vers le VPC
-   - D’un autre fournisseur vers votre VPC
+Les VPN de site-à-site offrent la possibilité d'interconnecter plusieurs VPCs, un bureau distant à un VPC, ou un autre fournisseur de cloud à un VPC. Un exemple d'un VPN site-à-site peut être trouvé dans l'article pratique [Créer un VPN site-à-site pour un VPC](../how-to/create-site-to-site-vpn-on-vpc.md).
 
-La configuration d’un VPN site-à-site est présentement indisponible sur le portail d’administration de CloudMC. Si vous avez besoin d’une telle configuration, contactez l'équipe de support de CloudMC, qui sera en mesure de vous aider.
+1. À partir du VPC cible, recherchez l'élément **VPNs site-à-site** et cliquez sur l'icône d'engrenage.
```
[the remark this answers is not in the document]
Comment (sweetsweetstefan) [reply fr/cloudstack-compute-service/working-with-vpcs.md:58]: Nope, it's definitely a gear sign :)

File: fr/cloudstack-compute-service/working-with-vpcs.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (sweetsweetstefan) [reply fr/cloudstack-compute-service/working-with-vpcs.md]: 57ffba3

File: fr/cloudstack-compute-service/working-with-vpcs.md [hunk 4]
```diff
@@ -4,58 +4,82 @@ slug: gestion-des-vpcs
...
+![Onglet de réseautique avec VPC](/assets/working-with-vpcs-3-fr.png)
+
+### Ajout d'un tier nouveau
+
+1. À partir du VPC cible, recherchez l'élément **Réseaux** et cliquez sur l'icône d'engrenage.
+1. Dans le coin supérieur droit, cliquez sur **Ajouter un réseau**.
+![Page de détails du VPC](/assets/working-with-vpcs-4-fr.png)
+1. Complétez le formulaire d'ajouter un réseau :
+   - **Nom :** Nom du tier (ex. *acme-net-web01*)
+   - **Description :** (Facultatif) Description du tier (ex. "Serveurs de web en production")
```
[the remark this answers is not in the document]
Comment (sweetsweetstefan) [reply fr/cloudstack-compute-service/working-with-vpcs.md]: 57ffba3

File: fr/cloudstack-compute-service/working-with-vpcs.md [hunk 5]
```diff
@@ -4,58 +4,82 @@ slug: gestion-des-vpcs
...
       - **Standard Tier :**  Réseau régulier qui supporte le NAT et la translation de port. Idéal pour les applications internes comme les serveurs de base de données.
-      - **Load Balanced Tier :**  Similar au "Standard Tier" mais offre en plus la possibilité de faire de la répartition de charge entre plusieurs instances déployées sur ce ties, via des règles de répartition de charge appliquées sur des adresse IP publiques. A noter que cette offre réseau ne peut s'appliquer qu'à un seul Tiers à l'intérieur d'un VPC.
-   1. **ACL :** Liste d’accès pour gérer les communications entre les tiers du VPC.
-      - **default_allow :**  Permet tout le traffic de/vers un autre tier du VPC.
+      - **Load Balanced Tier :**  (Par défaut) Similar au "Standard Tier" mais offre en plus la possibilité de faire de la répartition de charge entre plusieurs instances déployées sur ce tier, via des règles de répartition de charge appliquées sur des adresses IP publiques. **Avis : Cette offre réseau ne peut s'appliquer qu'à un seul tier à l'intérieur d'un VPC.**
+   - **Paserelle :**  L'adresse IP de la passerelle par défaut pour le tier de réseau à créer.
```
[the remark this answers is not in the document]
Comment (sweetsweetstefan) [reply fr/cloudstack-compute-service/working-with-vpcs.md]: 57ffba3

File: fr/cloudstack-compute-service/working-with-vpcs.md [hunk 6]
```diff
@@ -4,58 +4,82 @@ slug: gestion-des-vpcs
...
+   - **Masque de sous-réseau :**  Le masque de sous-réseau du tier réseau à créer.
```
[the remark this answers is not in the document]
Comment (sweetsweetstefan) [reply fr/cloudstack-compute-service/working-with-vpcs.md]: 57ffba3

File: fr/cloudstack-compute-service/working-with-vpcs.md (hunk 1 above)
Comment (sweetsweetstefan) [reply fr/cloudstack-compute-service/working-with-vpcs.md]: 57ffba3

**Pull request merged**